docs/source/highlights.md

```diff
@@ -46,6 +46,7 @@ transformations. These currently include, for example:
 - `Rand2DElastic`: Random elastic deformation and affine in 2D
 - `Rand3DElastic`: Random elastic deformation and affine in 3D
 
+[Medical transforms tutorial](https://github.com/Project-MONAI/MONAI/blob/master/examples/notebooks/transforms_demo_2d.ipynb) shows the detailed usage of several MONAI medical specific transforms.
```

Comment:
could be "2D transforms tutorial",  "MONAI medical specific transforms" -> "MONAI medical image specific transforms"

Reply:
updated in the latest comment.
Thanks.